lisa/operating_system.py

```diff
@@ -131,16 +170,47 @@ def _get_detect_string(cls, node: Any) -> Iterable[str]:
         cmd_result = typed_node.execute(cmd="cat /etc/SuSE-release", no_error_log=True)
         yield get_matched_str(cmd_result.stdout, cls.__suse_release_pattern)
 
+    def _get_os_version(self) -> OsVersion:
+        return self._os_version
+
 
 class Windows(OperatingSystem):
     def __init__(self, node: Any) -> None:
         super().__init__(node, is_posix=False)
+        self._get_os_version
+
+    def _get_os_version(self) -> OsVersion:
+        self._os_version.vendor = "Microsoft Corporation"
+        cmd_result = self._node.execute(
+            cmd="systeminfo | findstr /B /C:'OS Version'",
```

Comment:
I have a try, it looks only work with double quote, not single quote.

Reply:
fixed